app/controllers/tariffs_controller.rb

```diff
@@ -0,0 +1,23 @@
+class TariffsController < ApplicationController
+  def index
+    @tariff = Tariff.all
+  end
+  def new
+    @tariff = Tariff.new
+  end
+  def show
+    @tariff = Tariff.find(params[:id])
+  end
+  def edit
+    @tariff = Tariff.find(params[:id])
```

Comment:
@romigos duplicate code. You need create some method like 
```
def bundle_tariff
@tariff = Tariff.find(params[:id])
end

```

Reply:
@romigos and use it
```
def new
  bundle_tariff
end
```

Reply:
fixed